File: output/llama_4_maverick/avatar/algo_based_translation/python/java/failed_translation/codeforces_231_A.java

import java.util.Scanner;

public class codeforces_231_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int count = 0;
        for (int i = 0; i < n; i++) {
            String[] input = scanner.next().split(" ");
            int sum = 0;
            for (String num : input) {
                sum += Integer.parseInt(num);
            }
            if (sum > 1) {
                count++;
            }
        }
        System.out.println(count);
    }
}

File: output/llama_4_maverick/avatar/algo_based_translation/java/python/codeforces_231_A.py

count = 0
t = int(input())
while t > 0:
    a, b, c = map(int, input().split())
    if (a == 1 and b == 1) or (a == 1 and c == 1) or (b == 1 and c == 1) or (a == 1 and b == 1 and c == 1):
        count += 1
    t -= 1
print(count)

